app-frontend/src/app/components/colorCorrectScenes/colorCorrectScenes.controller.js

```diff
@@ -1,93 +1,46 @@
 export default class ColorCorrectScenesController {
     constructor( // eslint-disable-line max-params
-        $log, $scope, $q, projectService, layerService, $state
+        $log, $scope, $q, projectService, layerService, $state, mapService
     ) {
         'ngInject';
         this.projectService = projectService;
         this.layerService = layerService;
         this.$state = $state;
         this.$q = $q;
+        this.getMap = () => mapService.getMap('project');
     }
 
     $onInit() {
-        this.project = this.$state.params.project;
-        this.projectId = this.$state.params.projectid;
         // Internal bookkeeping to handle grid selection functionality.
         this.selectedTileX = null;
         this.selectedTileY = null;
-        this.gridHighlight = null;
         this.selectingScenes = false;
+        this.projectid = this.$state.params.projectid;
 
-        // Populate project scenes
-        if (!this.project) {
-            if (this.projectId) {
-                this.loading = true;
-                this.projectService.query({id: this.projectId}).then(
-                    (project) => {
-                        this.project = project;
-                        this.loading = false;
-                        this.populateSceneList();
-                    },
-                    () => {
-                        this.$state.go('library.projects.list');
-                    }
-                );
-            } else {
-                this.$state.go('library.projects.list');
-            }
-        } else {
-            this.populateSceneList();
-        }
-    }
-
-    $onChanges(changesObj) {
-        if (changesObj.clickEvent && changesObj.clickEvent.currentValue && this.mapZoom) {
-            this.selectGridCellScenes(changesObj.clickEvent.currentValue);
-        }
-    }
-
-    populateSceneList() {
-        // If we are returning from a different state that might preserve the
-        // sceneList, like the color correction adjustments, then we don't need
-        // to re-request scenes.
-        if (this.loading || this.sceneList.length > 0) {
-            this.layersFromScenes();
-            return;
-        }
-
-        delete this.errorMsg;
-        this.loading = true;
-        // save off selected scenes so you don't lose them during the refresh
-        this.sceneList = [];
-        this.projectService.getAllProjectScenes({projectId: this.project.id}).then(
-            (allScenes) => {
-                this.sceneList = allScenes;
-                this.layersFromScenes();
-            },
-            () => {
-                this.errorMsg = 'Error loading scenes.';
```

Comment:
Request status should be passed into the list views, now that the parent view takes care of the state / request